### ISSUE TITLE
Part 2: Update Address capture fields: I want to add the usual place of residence and a second address for a mother, deceased or informant

### ISSUE THREAD

**Issue by anny320** [opencrvs/opencrvs-core#2568]
### Description
- [Updates urban fields to Town, Street / Plot number, Postcode](https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=2243%3A47582)
- [Updates rural fields to just Village](https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=2243%3A47582)
- Removes yes/no for is the residential address the same as the mothers current address
![Screenshot 2022-03-01 at 15.01.50.png](https://images.zenhubusercontent.com/5e96a2d4befb239e4f2bce95/a40d7792-c4dd-4e7d-8f23-3f146dafec1a)
- Review page label for the first address should be ‘Usual place of residence’
- Rename "What is her place of origin (heritage)?" address to "2nd address" and place it at the bottom of the form (later we will configure this to be disabled by default)
- Create a "2nd address" for deceased and place it at the bottom of the form (later we will configure this to be disabled by default)
- For death informants we need to use the address structure as above for father.  Currently it is not using this structure. - refer to father ticket
- For birth informants we should use the address structure as above for mother

- Note. [Father’s usual place of residence](https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=2220%3A48002) is in another ticket. 

### Acceptance criteria

GIVEN I am an agent (Field agent, Reg agent or registrar)
WHEN navigate to the Mother’s Details, Deceased or informants
THEN show the address defaults to where i am assigned. 

GIVEN I am an agent (Field agent, Reg agent or registrar)
WHEN I select rural
THEN show the village input field

GIVEN I am on the review page
THEN the label for the address is ‘Usual place of residence’

### Design

[https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=2220%3A47829](https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=2220%3A47829)

### Dev-tasks

- [ ]  Task

**Comment by euanmillar** [comment]
Requires further research and dicsussion.

**Comment by anny320** [comment]
have added #2571 and split the ticket issues @euanmillar

**Comment by euanmillar** [comment]
Thanks @anny320 we will bring https://github.com/opencrvs/opencrvs-core/issues/2571 into planning and wait for the next discussion for this one.

**Comment by euanmillar** [comment]
@jpye-finch just waiting on the design asset we discussed for form flow
